releasenotes/notes/fix-houston-tc-mirred-bfb16c89f63b472a.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Adds a workaround for dmesg errors like:
```

Comment:
It would be helpful to name the playbook, and how/when to use it.

Reply:
done